scheduler/ipp.c

```diff
@@ -9374,6 +9374,9 @@ restart_job(cupsd_client_t  *con,	/* I - Client connection */
 		"Restarted by \"%s\" with job-hold-until=%s.",
                 username, attr->values[0].string.text);
     cupsdSetJobHoldUntil(job, attr->values[0].string.text, 0);
+    cupsdSetJobState(job, IPP_JOB_HELD, CUPSD_JOB_DEFAULT,
+		     "Job restarted by user with job-hold-until=%s",
+		     attr->values[0].string.text);
 
     cupsdAddEvent(CUPSD_EVENT_JOB_CONFIG_CHANGED | CUPSD_EVENT_JOB_STATE,
```

Comment:
and go ahead and remove this call since the SetJobState will send the event for us...

Reply:
Done.